addons/l10n_it_edi/models/account_edi_proxy_user.py

```diff
@@ -1,27 +1,425 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import fields, models
+import uuid
+import logging
+from lxml import etree
+from markupsafe import Markup
+
+from odoo import _, fields, models
+from odoo.addons.account_edi_proxy_client.models.account_edi_proxy_user import AccountEdiProxyError
+from odoo.addons.l10n_it_edi.tools.xml_utils import get_text, get_datetime, format_errors
+from odoo.exceptions import UserError
+from odoo.tools import html_escape as escape
```

Comment:
From what I understand, `tools.html_escape` is deprecated in favor of `markupsafe.escape()`.